docs/dictionary/keyword/else-if.lcdoc

```diff
@@ -0,0 +1,35 @@
+Name: else if
+
+Type: keyword
+
+Syntax: else if
+
+Summary:
+Used within an <if> <control structure> to evaluate an alternate <conditional expression|condition> and contain
```

Comment:
this should be `<condition|conditional expression>` - so that it refers to the condition parameter but is rendered as 'conditional expression'.